scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A script to check that travis.yml file & protractor.conf.js have the
+same test suites."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+import python_utils
+import utils
+
+# These 4 test suites are not present in travis ci.
+# One is extra ie. (full: [*.js]), and three other tests that
+# are being run by circleCi.
+TEST_SUITES_NOT_RUN_ON_TRAVIS = ['full', 'classroomPage', 'fileUploadFeatures',
+    'topicAndStoryEditor']
+
+
+def get_e2e_suite_names_from_jobs_travis_yml_file():
+    """Extracts the env/jobs section from the .travis.yml file.
+
+    Returns:
+        list(str). A list of names of test suites in sorted alphabetical
+        order of jobs section in travis.yml file.
+    """
+    travis_file_content = read_and_parse_travis_yml_file()
+    jobs_str = python_utils.convert_to_bytes(travis_file_content['env']['jobs'])
+    suites_from_jobs = []
+    # The following line extracts the test suite name from the jobs section
+    # that is in the form RUN_E2E_TESTS_ACCESSIBILITY=true.
+    test_regex = re.compile(r'RUN_E2E_TESTS_([a-zA-Z_-]*)=')
+    jobs = test_regex.findall(jobs_str)
+    for job in jobs:
+        suites_from_jobs.append(
+            utils.snake_case_to_camel_case(job.lower()))
+
+    return sorted(suites_from_jobs)
+
+
+def get_e2e_suite_names_from_script_travis_yml_file():
+    """Extracts the script section from the .travis.yml file.
+
+    Returns:
+        list(str). A list of names of test suites in sorted alphabetical
+        order of script section in travis.yml file.
+    """
+    travis_file_content = read_and_parse_travis_yml_file()
+    script_str = python_utils.convert_to_bytes(travis_file_content['script'])
+    # The following line extracts the test suites from patterns like
+    # bash scripts/run_e2e_tests.sh --suite="accessibility"
+    hyphen_between_regex = re.compile(
```

Comment:
The name of this is a bit weird. Maybe call it e2e_test_invocation_regex (or something that describes what the string it's matching represents conceptually).

Reply:
Done.